examples/data-sources/grafana_library_panel/bad-ExactlyOneOf.tf

```diff
@@ -0,0 +1,14 @@
+resource "grafana_library_panel" "test_bad_inputs" {
+  name       = "test name"
+  model_json = jsonencode({
+    gridPos  = {
+      h      = 8
+      w      = 12 }
```

Comment:
Can you run `terraform fmt` in this directory please? Thanks!

Reply:
should I align these `=` signs or leave as the formatter suggests? (see below)
<img width="1108" alt="image" src="https://user-images.githubusercontent.com/9123665/152364250-408c3259-df13-4c2d-93ab-1e981421e119.png">

Reply:
Just run `terraform fmt examples/data-sources/grafana_library_panel/` and it will make the pertinent changes. In #372 I'm doing the same for all the other directories containing `.tf` files.

Reply:
https://github.com/grafana/terraform-provider-grafana/pull/370/commits/89cfdaeeb6adc38568c06e2b6ea36c7381c95188